include/mixer.h

```diff
@@ -79,26 +79,29 @@ class MixerChannel {
 	void AddStretched(Bitu len,Bit16s * data);		//Strech block up into needed data
 
 	void FillUp(void);
-	void Enable(bool _yesno);
+	void Enable(bool should_enable);
+	void FlushSamples();
 
-	float          volmain[2];
-	MixerChannel*  next;
-	const char*    name;
-	Bitu           done;           //Timing on how many samples have been done by the mixer
-	bool           enabled;
+	float volmain[2] = {0.0f};
+	MixerChannel *next = nullptr;
+	const char *name = nullptr;
+	Bitu done = 0u; // Timing on how many samples have been done by the mixer
+	bool isEnabled = false;
```

Comment:
I suggest renaming the field `is_enabled` - it will be consistent with other fields in this class (and generally in code).

Reply:
Sounds good!